app-starter/components/AppSidebar.vue

```diff
@@ -63,6 +73,32 @@ export default {
           duration: this.scrollDuration
         });
       }
+    },
+    /**
+     * Resize the sidebar, if the 'resizable' option is enabled.
+     */
+    onResize () {
+      const me = this;
+      const sidebarEl = me.$refs.sidebar.$el;
+      sidebarEl.style.transition = 'initial';
+
+      // Resize on mouse move
+      function onMouseMove (e) {
```

Comment:
can we use a arrow function here instead?
then we do not need to have the `me` variable, because `this` can be used in the function directly

Reply:
To my knowledge declaring an arrow function and inlining it to the event handler only works, if the event doesnt have to be unregistered. So I could do it for `onMouseUp ` but not for `onMouseMove`. For clarity I decided to declare both handlers explicitly. 
I can give it a try though.

Reply:
Ok my bad. Obviously it works as well with arrow functions. You just can't inline them. Want to give it a quick test/ approval again before I merge?